localenv/mock-account-servicing-entity/app/components/Button.tsx

```diff
@@ -0,0 +1,46 @@
+import { cva, cx, type VariantProps } from 'class-variance-authority'
+import { forwardRef } from 'react'
+import { ButtonOrLink, type ButtonOrLinkProps } from './ButtonOrLink'
+
+const buttonStyles = cva(
+  'inline-flex items-center justify-center focus:outline-none disabled:cursor-not-allowed',
+  {
+    variants: {
+      intent: {
+        default:
+          'bg-[#F37F64] hover:bg-[#DA725A] disabled:bg-mercury disabled:text-gray-500 shadow-md text-white',
+        danger:
+          'disabled:bg-red-200 bg-red-500 hover:bg-red-600 shadow-md text-white'
+      },
+      size: {
+        sm: 'px-2 py-1 rounded-md font-medium',
```

Comment:
you could add rounded and font size to general styles, as they are the same for both sizes